WORK_SUMMARY_2025-10-20.md

```
@@ -0,0 +1,777 @@
+  "success": true,
+  "job_posting_id": 3,
+  "card_id": 12,
+  "matching_vector_id": null,
+  "card": {...},
+  "matching_texts": {...},
+  "warning": "Matching vectors generation failed (OpenAI API error)"
+}
+```
+
+### 로그 출력 예시
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)